src/services/timer.js

```diff
@@ -43,3 +43,32 @@ export const determineNextPeriod = (currentPeriod, counter) => {
 
   return PERIODS.work;
 };
+export const displyNotification = (nextPeriod, currentPeriod) => {
+  const notificatinFunc = (text, icon) => new Notification('Pomodoro', {
+    body: text,
+    icon,
+  });
+  const showNotification = () => {
```

Comment:
You don't need to define this function inside the `displayNotification` function. You should define it outside.

Reply:
@bah3wee96 you still haven't addressed my comment here. Also, you still haven't fixed the spelling mistake.